### PR TITLE
Phase2-gem49 Use constants defined in DataFormats

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Use constants defined in the DetId classes for GEM and ME0

#### PR validation:

Tested using scripts in Geometry/GEMGeometryBuilder/test/python

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/GEMGeometryBuilder/src/GEMGeometryBuilder.cc [hunk 1]
```diff
@@ -3,9 +3,9 @@
 
  Description: GEM Geometry builder from DD and DD4HEP
               DD4hep part added to the original old file (DD version) made by M. Maggi (INFN Bari)
-//
-// Author:  Sergio Lo Meo (sergio.lo.meo@cern.ch) following what Ianna Osburne made for DTs (DD4HEP migration)
-//          Created:  27 Jan 2020 
+              Sergio Lo Meo (sergio.lo.meo@cern.ch) following what Ianna Osburne made for DTs (DD4HEP migration)
```
Comment (cvuosalo) [inline Geometry/GEMGeometryBuilder/src/GEMGeometryBuilder.cc:6]:
```suggestion
              Sergio Lo Meo (sergio.lo.meo@cern.ch) following what Ianna Osborne made for DTs (DD4HEP migration)
```

Comment (cvuosalo) [reply Geometry/GEMGeometryBuilder/src/GEMGeometryBuilder.cc:6]: ;-)

**Comment by cvuosalo** [comment]
+1

**Comment by kpedro88** [comment]
+upgrade

**Comment by silviodonato** [comment]
+1

**Pull request merged**